src/components/network/network-map.js

```diff
@@ -336,6 +332,7 @@ const NetworkMap = (props) => {
                 onHover: ({ object, x, y }) => {
                     setCursorType(object ? 'pointer' : 'grab');
                 },
+                getNameOrId: getNameOrId,
```

Comment:
If we pass `getNameOrId` in `SubstationLayer` which changes each time the parameter `useName` changes then maybe we could avoid tu pass `useName` as props in `SubstationLayer` and trigger label refresh with `getNameOrId`